staging/src/k8s.io/kube-aggregator/pkg/client/clientset_generated/clientset/typed/apiregistration/v1/apiservice.go

```diff
@@ -57,20 +58,27 @@ type APIServiceInterface interface {
 type aPIServices struct {
 	client  rest.Interface
 	clients []rest.Interface
+	te      string
 }
 
 // newAPIServices returns a APIServices
 func newAPIServices(c *ApiregistrationV1Client) *aPIServices {
+	return newAPIServicesWithMultiTenancy(c, "system")
+}
+
+func newAPIServicesWithMultiTenancy(c *ApiregistrationV1Client, tenant string) *aPIServices {
 	return &aPIServices{
 		client:  c.RESTClient(),
 		clients: c.RESTClients(),
+		te:      tenant,
```

Comment:
same here te --> tenant

Reply:
Same as above, will refactor it in another PR.